src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -0,0 +1,121 @@
+import { generateConfluenceMarkdown } from '../Utilities/generateConfluenceMarkdown';
+import { Issue } from '../issueFinder';
+import { DateTime, Duration } from 'luxon';
+
+const eightDaysAgo = DateTime.utc().minus(Duration.fromISO('P8D')).toISO();
+const twoDaysAgo = DateTime.utc().minus(Duration.fromISO('P2D')).toISO();
+const fakeIssues: Issue[] = [
+    {
+        title: 'Formatting seems to use Prettier options by default',
+        createdAt: eightDaysAgo,
+        repositoryNameWithOwner: 'bc/typescript',
+        url: 'https://github.com/bc/typescript/issues/30',
+        updatedAt: '',
+        labels: ['help wanted', 'documentation'],
+    },
+
+    {
+        title: 'style config',
+        createdAt: eightDaysAgo,
+        repositoryNameWithOwner: 'material-ui/mui',
+        url: 'https://github.com/material-ui/issues/24',
+        updatedAt: '',
+        labels: ['good first issue', 'help wanted', 'documentation'],
+    },
+];
+
+const singleIssue: Issue[] = [
+    {
+        title: 'ToC: links to markdown headings',
+        createdAt: eightDaysAgo,
+        repositoryNameWithOwner: 'marmelab/react-admin',
+        url: 'https://github.com/marmelab/react-admin/issues/5620',
+        updatedAt: twoDaysAgo,
+        labels: ['good first issue', 'documentation'],
+    },
+];
+
+describe('generateConfluenceMarkdown function', () => {
+    it('should not return a dependency with issue information', () => {
+        const mapWithoutIssues: Map<string, Issue[]> = new Map();
+        const noIssues: Issue[] = [];
+        const dependency = 'TestDependency';
+
+        const now = DateTime.local();
+        const createdAt = DateTime.fromISO(singleIssue[0].createdAt);
+        const ageInDays = now.diff(createdAt, 'days').days;
+        const ageInWholeDays = Math.round(ageInDays);
+
+        const oneDependency = mapWithoutIssues.set(dependency, noIssues);
+        const results = generateConfluenceMarkdown(oneDependency);
+        expect(results).not.toContain(`h3. ${dependency}`);
+        expect(results).not.toContain('\n ||*Title*||*Age*||');
+        expect(results).not.toContain(`|[${fakeIssues[0].title}|${fakeIssues[0].url}]|`);
```

Comment:
Since we didn't pass in any issues, I'm not sure we need to check this level of detail. Maybe just expect not to contain the word 'days' would be enough. (In addition to not containing the h3 and  title/age header.)